src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -619,7 +618,7 @@ def fit_velocity(peak_times, channel_dist):
 
 def get_velocity_above(template, channel_locations, sampling_frequency, **kwargs):
     """
-    Compute the velocity above the max channel of the template.
+    Compute the velocity above the max channel of the template in units (unit of channel locations) per second, usually um/s.
```

Comment:
```suggestion
    Compute the velocity above the max channel of the template in um/s.
```
This is always the case. The convention is that channel_locations are in um. Maybe we should make this clearer in the docs